src/components/GettingStartedCard/GettingStartedCard.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import {
+  Button,
+  Card,
+  CardActions,
+  CardBody,
+  CardHeader,
+  CardTitle,
+  Split,
+  SplitItem,
+} from '@patternfly/react-core';
+import CloseIcon from '@patternfly/react-icons/dist/js/icons/close-icon';
+import { useLocalStorage } from '../../hooks';
+
+type GettingStartedCardProps = {
+  localStorageKey: string;
+  title: string;
+  imgSrc: string;
+};
+
+const LOCAL_STORAGE_KEY = 'getting-started-card';
+
+export const GettingStartedCard: React.FC<GettingStartedCardProps> = ({
+  localStorageKey,
+  title,
+  imgSrc,
+  children,
+}) => {
+  const [storageKeys, setStorageKeys] =
+    useLocalStorage<{ [key: string]: boolean }>(LOCAL_STORAGE_KEY);
+
+  const keys = typeof storageKeys === 'object' ? storageKeys : {};
+  const isDismissed = keys[localStorageKey];
+
+  return (
+    !isDismissed && (
+      <Card style={{ margin: '15px' }}>
```

Comment:
Remove the inline style and wrap the card with a `<PageSection>` to control the margins?  I think we want the card's context to control the spacing.